code/_onclick/hud/human.dm

```diff
@@ -449,6 +464,10 @@
 		if(H.r_store)
 			H.r_store.screen_loc = ui_storage2
 			H.client.screen += H.r_store
+		if(H.neck)
+			H.neck.screen_loc = ui_neck
+			H.client.screen += H.neck
+
 	else
```

Comment:
Don't need anything in this proc, since it's for the 'Persistent inventory', eg the stuff that's never gets hidden.
```suggestion
		if(H.r_store)
			H.r_store.screen_loc = ui_storage2
			H.client.screen += H.r_store
	else
```